pytorch_lightning/plugins/checkpoint/checkpoint.py

```diff
@@ -0,0 +1,24 @@
+from abc import ABC, abstractmethod
+from pathlib import Path
+from typing import Any, Dict, Union
+
+
+class CheckpointIOPlugin(ABC):
+    @abstractmethod
+    def save_checkpoint(self, checkpoint: Dict[str, Any], path: Union[str, Path]) -> None:
+        """Save model/training states as a checkpoint file through state-dump and file-write.
+
+        Args:
+            checkpoint: dict containing model and trainer state
+            path: write-target path
+        """
+
+    @abstractmethod
+    def load_checkpoint_file(self, path: Union[str, Path]) -> Dict[str, Any]:
```

Comment:
```suggestion
    def load_checkpoint(self, path: Union[str, Path], storage_options: Optional[Mapping] = None) -> Dict[str, Any]:
```
for consistency with above: load/save checkpoint